### DISCUSSION TITLE
Help in assigning roles

### DISCUSSION THREAD

**Discussion by Hrsh3112** [danthe1st/Mee6-bypasser#39] [Support]
Even after configuring the bot, it still does not assign roles as per setup and it's a problem as one of my big server runs on it for levels through mee6 please fix the issue it's really problematic

**Comment by Hrsh3112** [comment]
> Even after configuring the bot, it still does not assign roles as per setup and it's a problem as one of my big server runs on it for levels through mee6 please fix the issue it's really problematic

It has all perms so, that's not a problem

**Comment by danthe1st** [comment]
What is the output of `mb!show`? Does it display the roles to assign?

Note that roles are only assigned when explicitely executing `!rank` and that this only happens once per minute.

**Reply by Hrsh3112** [reply]
When i type mb!show, it shows the role to assign, i tried using !rank with my main and a alt account too, it didnt work, no roles were assigned

**Reply by danthe1st** [reply]
Can you show an example that includes the output of `mb!show`and one that shows how that the highest role of Mee6-bypasser is higher than the role to assign? Does something happen when the server owner uses `!rank`?

**Reply by danthe1st** [reply]
Also, please try again. Does it work, now?
